defaults/main.yml

```diff
@@ -10,8 +10,8 @@ storage_pool_defaults:
   type: lvm
 
   encryption: false
-  encryption_passphrase: null
-  encryption_key_file: null
+  password: null
```

Comment:
To me it seems that the `encryption_` prefix should be treated as a grouping mechanism like using a dictionary `encryption: ...`, and therefore `encryption_password` and `encryption_key` seem to make more sense here, for me.

Reply:
That is correct. We use prefixes for a sort of namespacing since there are so many parameters.